frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -17,27 +17,31 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
   const errorMessage = (() => {
     if (!error) return undefined;
     switch (error) {
-      // Invalid interpretation
+      // Invalid ballot interpretations
       case 'invalid_test_mode':
-        return isTestMode ? 'Live Ballot detected.' : 'Test ballot detected.';
+        return isTestMode
+          ? 'Live ballot detected. Scanner is in test mode.'
+          : 'Test ballot detected. Scanner is in live mode.';
       case 'invalid_election_hash':
-        return 'Scanned ballot does not match the election this scanner is configured for.';
+        return 'The ballot does not match the election this scanner is configured for.';
       case 'invalid_precinct':
-        return 'Scanned ballot does not match the precinct this scanner is configured for.';
+        return 'The ballot does not match the precinct this scanner is configured for.';
       case 'unreadable':
-        return 'There was a problem reading this ballot. Please try again.';
       case 'unknown':
-        return undefined;
-      // Precinct scanner error
+        return 'There was a problem reading this ballot. Please scan again.';
+      // Precinct scanner errors
       case 'both_sides_have_paper':
+        return 'Scanner detected unexpected ballots in both the front and back. Remove both ballots to continue.';
       case 'paper_in_front_on_startup':
+        return 'Scanner detected an unexpected ballot in the tray. Remove ballot to continue.';
       case 'paper_in_back_on_startup':
       case 'paper_in_back_after_accept':
-        return 'Take your ballot out of the tray and try again.';
+        return 'Scanner detected an unexpected ballot at the back of the scanner. Remove ballot to continue.';
       case 'unexpected_paper_status':
+        return 'Scanner detected an unexpected ballot in scanner. Remove ballot to continue.';
       case 'unexpected_event':
       case 'plustek_error':
-        return undefined;
+        return 'The scanner experienced an error and needs to be reset. Please turn off and unplug the scanner from the power outlet. Plug it back in and turn it on again.';
```

Comment:
We're still working out what actually will be needed to recover from these cases. Currently, we're experimenting with the scan service automatically disconnecting and reconnecting to Plustek to try to mitigate this without any need for user action (with a 5s delay).

If we stick with that, then maybe this should just have a spinner or something? And then have a timeout in the frontend where if this screen shows for more than 5s we switch to something like the error message you have here.

Now that I'm thinking about it, I wonder if just asking for a manual reset might be clearer than having this automatic recovery. Worth thinking through

Reply:
So then, is this safe to commit in the meantime?

Reply:
Sure!